services/core-web/src/tests/actionCreators/staticContentActionCreator.spec.js

```diff
@@ -232,159 +232,159 @@ describe("`fetchVarianceDocumentCategoryOptions` action creator", () => {
       expect(dispatch).toHaveBeenCalledTimes(3);
     });
   });
   it("Request failure, dispatches `error` with correct response", () => {
     mockAxios.onGet(url, MOCK.createMockHeader()).reply(400, MOCK.ERROR);
-    return fetchNoticeOFWorkApplicationDocumentTypeOptions()(dispatch).then(() => {
+    return fetchNoticeOfWorkApplicationDocumentTypeOptions()(dispatch).then(() => {
       expect(requestSpy).toHaveBeenCalledTimes(1);
       expect(errorSpy).toHaveBeenCalledTimes(1);
       expect(dispatch).toHaveBeenCalledTimes(2);
     });
   });
 });
 
-describe("`fetchNoticeOFWorkUndergroundExplorationTypeOptions` action creator", () => {
+describe("`fetchNoticeOfWorkUndergroundExplorationTypeOptions` action creator", () => {
   const url = ENVIRONMENT.apiUrl + API.NOW_UNDERGROUND_EXPLORATION_TYPE_OPTIONS;
   it("Request successful, dispatches `success` with correct response", () => {
     const mockResponse = { data: { success: true } };
     mockAxios.onGet(url).reply(200, mockResponse);
-    return fetchNoticeOFWorkUndergroundExplorationTypeOptions()(dispatch).then(() => {
+    return fetchNoticeOfWorkUndergroundExplorationTypeOptions()(dispatch).then(() => {
       expect(requestSpy).toHaveBeenCalledTimes(1);
       expect(successSpy).toHaveBeenCalledTimes(1);
       expect(dispatch).toHaveBeenCalledTimes(3);
     });
   });
   it("Request failure, dispatches `error` with correct response", () => {
     mockAxios.onGet(url, MOCK.createMockHeader()).reply(400, MOCK.ERROR);
-    return fetchNoticeOFWorkUndergroundExplorationTypeOptions()(dispatch).then(() => {
+    return fetchNoticeOfWorkUndergroundExplorationTypeOptions()(dispatch).then(() => {
       expect(requestSpy).toHaveBeenCalledTimes(1);
       expect(errorSpy).toHaveBeenCalledTimes(1);
       expect(dispatch).toHaveBeenCalledTimes(2);
     });
   });
 });
 
-describe("`fetchNoticeOFWorkApplicationProgressStatusCodes` action creator", () => {
+describe("`fetchNoticeOfWorkApplicationProgressStatusCodes` action creator", () => {
   const url = ENVIRONMENT.apiUrl + API.NOW_APPLICATION_PROGRESS_STATUS_CODES;
   it("Request successful, dispatches `success` with correct response", () => {
     const mockResponse = { data: { success: true } };
     mockAxios.onGet(url).reply(200, mockResponse);
-    return fetchNoticeOFWorkApplicationProgressStatusCodes()(dispatch).then(() => {
+    return fetchNoticeOfWorkApplicationProgressStatusCodes()(dispatch).then(() => {
       expect(requestSpy).toHaveBeenCalledTimes(1);
       expect(successSpy).toHaveBeenCalledTimes(1);
       expect(dispatch).toHaveBeenCalledTimes(3);
     });
   });
   it("Request failure, dispatches `error` with correct response", () => {
     mockAxios.onGet(url, MOCK.createMockHeader()).reply(400, MOCK.ERROR);
-    return fetchNoticeOFWorkApplicationProgressStatusCodes()(dispatch).then(() => {
+    return fetchNoticeOfWorkApplicationProgressStatusCodes()(dispatch).then(() => {
```

Comment:
Thanks for fixing all of these... I don't know how I managed to case 'OF' incorrectly in every situation.